src/app/forms/templates/general/hgv-trl-body.template.ts

```diff
@@ -0,0 +1,53 @@
+import { ValidatorNames } from '@forms/models/validators.enum';
+import { FormNode, FormNodeTypes } from '../../services/dynamic-form.types';
+
+export const hgvAndTrlBodyTemplate: FormNode = {
+  name: 'bodySection',
+  label: 'Body',
+  type: FormNodeTypes.GROUP,
+  children: [
+    {
+      name: 'make',
+      label: 'Body make',
+      value: '',
+      type: FormNodeTypes.CONTROL,
+      validators: [{ name: ValidatorNames.MaxLength, args: 30 }, { name: ValidatorNames.Required }]
+    },
+    {
+      name: 'model',
+      label: 'Body model',
+      value: '',
+      type: FormNodeTypes.CONTROL,
+      validators: [{ name: ValidatorNames.MaxLength, args: 30 }, { name: ValidatorNames.Required }]
+    },
+    {
+      name: 'bodyType',
+      label: 'Body type',
+      value: '',
+      type: FormNodeTypes.GROUP,
+      children: [
+        {
+          name: 'description',
+          label: 'Body type',
+          value: '',
+          type: FormNodeTypes.CONTROL,
+          validators: [{ name: ValidatorNames.Required }]
+        },
+      ]
+    },
+    {
+      name: 'functionCode',
+      label: 'Function code',
+      value: '',
+      type: FormNodeTypes.CONTROL,
+      validators: [{ name: ValidatorNames.MaxLength, args: 1 }]
```

Comment:
Just looked at the service design and function code seems to be a dropdown, might be worth seeing if it's intentional or not

Reply:
I have used the service design and made it a dropdown already 🙂